lib/foundation_rails_helper/form_builder.rb

```diff
@@ -187,20 +181,27 @@ def calculate_input_size(prefix_options, postfix_options)
     end
 
     def wrap_prefix_and_postfix(block, prefix_options, postfix_options)
-      prefix = tag_from_options('prefix', prefix_options)
-      postfix = tag_from_options('postfix', postfix_options)
+      prefix = tag_from_options("prefix", prefix_options)
+      postfix = tag_from_options("postfix", postfix_options)
```

Comment:
Please go back to single quotes because that fits the style guide we are implementing (#119).